src/os/bluestore/BlueFS.cc

```diff
@@ -901,6 +921,19 @@ int BlueFS::_read(
            << " 0x" << std::hex << off << "~" << len << std::dec
 	   << " from " << h->file->fnode << dendl;
 
+  logger->inc(l_bluefs_reads);
+  logger->inc(l_bluefs_read_bytes, len);
+  if (h->file->contents.length()) {
```

Comment:
OK, the latter isn't possible due to assert(h->file->contents.length() == offset + length); in _flush_range